### PR TITLE
doc updates

### PR DESCRIPTION
**Pull request by JaydenLiang**
target version: 3.3.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,55 @@
...
+| fortigate-autoscale-azure-funcapp.zip             | Source code for the FortiGate Autoscale handler - Azure function.      |
+| fortigate-autoscale-azure-template-deployment.zip | Azure template. Use this to deploy the solution on the Azure platform. |
+
+## Deployment guide
+A deployment guide is available from the Fortinet Document Library:
+
+  + [ FortiGate / FortiOS 6.4 / Deploying auto scaling on Azure](https://docs.fortinet.com/document/fortigate-public-cloud/6.4.0/azure-administration-guide/161167/deploying-auto-scaling-on-azure)
+
+## Launch a demo
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Ffortinet%2Ffortigate-autoscale%2Fmaster%2Fazure_template_deployment%2Ftemplates%2Fdeploy_fortigate_autoscale.hybrid_licensing.json" target="_blank"><img src="http://azuredeploy.net/deploybutton.png"/></a>
```
Comment (klc178) [inline README.md:38]: Does this need to be updated for the 3.3.0 template?

File: README.md [hunk 2]
```diff
@@ -1,54 +1,55 @@
...
+
+
+## Project development history
+| Version | Details | Documentation |
+| ------- | ------- | ------------- |
+| 3.3.0 (latest) | FortiAnalyzer integration is available with this version.|
```
Comment (klc178) [inline README.md:44]:
```suggestion
| 3.3.0 (latest) | Added support for FortiAnalyzer integration.|
```

File: templates/deploy_fortigate_autoscale.hybrid_licensing.json [hunk 3]
```diff
@@ -6,20 +6,20 @@
...
             }
         },
         "AdminPassword": {
             "type": "SecureString",
             "metadata": {
-                "description": "FortiGate administrator password on all VMs as well as the FortiAnalyzer if FortiAnalyzer integration enabled. This field must be between 11 and 26 characters and must include at least one uppercase letter, one lowercase letter, one digit, and one special character such as (! @ # $ %)."
+                "description": "FortiGate administrator password on all VMs as well as the FortiAnalyzer if 'FortiAnalyzer integration' is enabled. Must be between 11 and 26 characters and must include at least one uppercase letter, one lowercase letter, one digit, and one special character such as (! @ # $ %)."
```
Comment (klc178) [inline templates/deploy_fortigate_autoscale.hybrid_licensing.json:15]:
```suggestion
                "description": "FortiGate administrator password on all VMs as well as the FortiAnalyzer if FortiAnalyzer integration is enabled. Must be between 11 and 26 characters and must include at least one uppercase letter, one lowercase letter, one digit, and one special character such as (! @ # $ %)."
```

File: templates/deploy_fortigate_autoscale.hybrid_licensing.json [hunk 4]
```diff
@@ -179,7 +179,7 @@
             "defaultValue": "",
             "type": "String",
             "metadata": {
-                "description": "The name of the Network Security Group associated with the subnets in the VNet. Required when using an existing VNet. The value should match the name of the existing Network Security Group associated with the subnets in the VNet. When creating a new VNet, you may specify a name for the Network Security Group. If left empty a name will be generated."
+                "description": "Name of the Network Security Group associated with the subnets in the VNet. Required when using existing VNet. The value should match the name of the existing Network Security Group associated with the subnets in the VNet. When creating a new VNet, you may specify a name for the Network Security Group. If left empty, a name will be generated."
```
Comment (klc178) [inline templates/deploy_fortigate_autoscale.hybrid_licensing.json:182]:
```suggestion
                "description": "Name of the Network Security Group associated with the subnets in the VNet. Required when using an existing VNet. The value should match the name of the existing Network Security Group associated with the subnets in the VNet. When creating a new VNet, you may specify a name for the Network Security Group. If left empty, a name will be generated."
```

File: README.md [hunk 5]
```diff
@@ -1,54 +1,55 @@
...
+A collection of **Node.js** modules and cloud-specific templates which support auto scaling functionality for groups of FortiGate-VM instances on various cloud platforms.
+
+This project contains the code and templates for the **Microsoft Azure** deployment. For deployment on other cloud platforms, visit the relevant repository:
+* The **AliCloud** deployment is in the  [alicloud-autoscale](https://github.com/fortinet/alicloud-autoscale/) repository.
+* The **Amazon AWS** deployment is in the [fortigate-autoscale-aws](https://github.com/fortinet/fortigate-autoscale-aws) repository.
+* The **GCP** deployment is in the [fortigate-autoscale-gcp](https://github.com/fortinet/fortigate-autoscale-gcp) repository.
+
+This project is organized in separate node modules:
+
+ * [fortigate-autoscale/core](core) contains the core logic and provides an interface that can be extended to deal with the differences in cloud platform APIs.
```
Comment (klc178) [inline README.md:12]: fortigate-autoscale-azure/core?

File: README.md [hunk 6]
```diff
@@ -1,54 +1,55 @@
...
+ * [fortigate-autoscale/azure](azure) contains an implementation for the **Microsoft Azure** platform API and **Cosmos DB** storage backend.
```
Comment (klc178) [inline README.md:13]: Please confirm the location.

File: README.md [hunk 7]
```diff
@@ -1,54 +1,55 @@
...
+
+The project also contains a deployment script that can generate packages for each cloud service's *serverless* implementation.
+
+## Deployment packages
+To generate local deployment packages:
+
+  1. From the [project release page](https://github.com/fortinet/fortigate-autoscale/releases), download the source code (.zip or .tar.gz) for the latest 2.0 version.
```
Comment (klc178) [inline README.md:20]: fortigate-autoscale-azure?

File: templates/deploy_fortigate_autoscale.hybrid_licensing.json [hunk 8]
```diff
@@ -263,85 +263,85 @@
...
+                "description": "IP address space of the VNet in CIDR notation. E.g. 10.0.0.0/16. Required when using an existing VNet; the value should match the address space of the target VNet."
             }
         },
         "VnetDeploymentMethod": {
             "defaultValue": "create new",
             "allowedValues": ["create new", "use existing"],
             "type": "String",
             "metadata": {
-                "description": "Options for Virtual Network (VNet) deployment. For requirements when using an existing resource group, refer to the documentation."
+                "description": "Options for Virtual Network (VNet) deployment. Note: The VNet resource group (specified in the 'VNet Resource Group Name' parameter) must be in the same region as the Autoscale resource group (specified in the 'Resource group' parameter). For requirements when using an existing VNet, refer to the documentation."
```
Comment (klc178) [inline templates/deploy_fortigate_autoscale.hybrid_licensing.json:330]: Please confirm the change in the last sentence (resource group to VNet).

File: templates/deploy_fortigate_autoscale.hybrid_licensing.json [hunk 9]
```diff
@@ -263,85 +263,85 @@
...
             }
         },
         "VnetName": {
             "defaultValue": "",
             "type": "String",
             "metadata": {
-                "description": "Name of the Azure VNet to connect to FortiGate Autoscale. Required when using an existing VNet. When creating a new VNet, this parameter can be left empty and a name will be generated."
+                "description": "Name of the Azure VNet  to connect to FortiGate Autoscale. Required when using an existing VNet. When creating a new VNet, this parameter can be left empty and a name will be generated."
```
Comment (klc178) [inline templates/deploy_fortigate_autoscale.hybrid_licensing.json:337]:
```suggestion
                "description": "Name of the Azure VNet to connect to FortiGate Autoscale. Required when using an existing VNet. When creating a new VNet, this parameter can be left empty and a name will be generated."
```

**Review by klc178 — CHANGES_REQUESTED** [review]
Fixed a few things and added some questions.

File: README.md (hunk 5 above)
Comment (JaydenLiang) [reply README.md:12]: the content [fortigate-autoscale/core] is correct.

File: README.md (hunk 6 above)
Comment (JaydenLiang) [inline README.md:13]:
```suggestion
 * [fortigate-autoscale-azure](https://github.com/fortinet/fortigate-autoscale-azure) contains an implementation for the **Microsoft Azure** platform API and **Cosmos DB** storage backend.

```

File: README.md (hunk 5 above)
Comment (JaydenLiang) [inline README.md:12]:
```suggestion
 * [autoscale-core](https://github.com/fortinet/autoscale-core) contains the core logic and provides an interface that can be extended to deal with the differences in cloud platform APIs.

```

File: README.md (hunk 7 above)
Comment (JaydenLiang) [inline README.md:20]:
```suggestion
  1. From the [project release page](https://github.com/fortinet/fortigate-autoscale-azure/releases), download the fortigate-autoscale-azure.zip for the latest versioin.

```

File: README.md [hunk 10]
```diff
@@ -1,54 +1,55 @@
...
+  2. Extract the source code.
+  3. Run `npm run build` at the project root directory.
+
+Deployment packages as well as source code will be available in the **dist** directory.
+
+| Package Name                                      | Description                                                            |
+| ------------------------------------------------- | ---------------------------------------------------------------------- |
+| fortigate-autoscale.zip                           | Source code for the entire project.                                    |
+| fortigate-autoscale-azure-funcapp.zip             | Source code for the FortiGate Autoscale handler - Azure function.      |
+| fortigate-autoscale-azure-template-deployment.zip | Azure template. Use this to deploy the solution on the Azure platform. |
```
Comment (JaydenLiang) [inline README.md:30]:
```suggestion

```

File: README.md (hunk 1 above)
Comment (JaydenLiang) [inline README.md:38]:
```suggestion
<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Ffortinet%2Ffortigate-autoscale-azure%2F3.3.0-rc.1%2Ftemplates%2Fdeploy_fortigate_autoscale.hybrid_licensing.json" target="_blank"><img src="http://azuredeploy.net/deploybutton.png"/></a>

```

File: README.md [hunk 11]
```diff
@@ -1,54 +1,55 @@
...
+| 2.0 | Added support for Hybrid Licensing (any combination of BYOL and/or PAYG instances). | A PDF for AWS is available in the 2.0 branch.<br/>[deploying-auto-scaling-on-aws-2.0.9.pdf](https://github.com/fortinet/fortigate-autoscale/blob/2.0/docs/deploying-auto-scaling-on-aws-2.0.9.pdf) |
+| 1.0 | Supports auto scaling for PAYG instances only.  | PDFs for AWS and for Azure are available in the 1.0.6 branch.<br/>[deploying-auto-scaling-on-aws-1.0.pdf](https://github.com/fortinet/fortigate-autoscale/blob/1.0/docs/deploying-auto-scaling-on-aws-1.0.pdf)<br/>[deploying-auto-scaling-on-azure-1.0.pdf](https://github.com/fortinet/fortigate-autoscale/blob/1.0/docs/deploying-auto-scaling-on-azure-1.0.pdf) |
+
+# Support
+Fortinet-provided scripts in this and other GitHub projects do not fall under the regular Fortinet technical support scope and are not supported by FortiCare Support Services.
+For direct issues, please refer to the [Issues](https://github.com/fortinet/fortigate-autoscale/issues) tab of this GitHub project.
```
Comment (JaydenLiang) [inline README.md:51]:
```suggestion
For direct issues, please refer to the [Issues](https://github.com/fortinet/fortigate-autoscale-azure/issues) tab of this GitHub project.

```

**Review by jamie-pate — APPROVED** [review]

File: README.md [hunk 12]
```diff
@@ -1,54 +1,50 @@
...
+
+ * [fortigate-autoscale-azure](https://github.com/fortinet/fortigate-autoscale-azure) contains an implementation for the **Microsoft Azure** platform API and **Cosmos DB** storage backend.
+
+
+The project also contains a deployment script that can generate packages for each cloud service's *serverless* implementation.
+
+## Deployment packages
+To generate local deployment packages:
+
+  1. From the [project release page](https://github.com/fortinet/fortigate-autoscale-azure/releases), download the fortigate-autoscale-azure.zip for the latest versioin.
```
Comment (klc178) [inline README.md:22]:
```suggestion
1. From the [project release page](https://github.com/fortinet/fortigate-autoscale-azure/releases), download the fortigate-autoscale-azure.zip for the latest version.
2. ```

**Review by klc178 — CHANGES_REQUESTED** [review]
Please update line 58 (description for FortiAnalyzer Instance Type) to:
"description": "Size of the FortiAnalyzer-VM. Not all instance types are supported. Review the documentation prior to selecting an instance."

**Comment by JaydenLiang** [comment]
> Please update line 58 (description for FortiAnalyzer Instance Type) to:
> "description": "Size of the FortiAnalyzer-VM. Not all instance types are supported. Review the documentation prior to selecting an instance."

done

**Review by klc178 — APPROVED** [review]

**Pull request merged**